dnd5e.mjs

```diff
@@ -578,6 +578,8 @@ Hooks.on("getItemDirectoryEntryContext", documents.Item5e.addDirectoryContextOpt
 
 Hooks.on("renderJournalPageSheet", applications.journal.JournalSheet5e.onRenderJournalPageSheet);
 
+Hooks.on("renderActiveEffectConfig", (app, html, data) => documents.ActiveEffect5e.onRenderActiveEffectConfig(html));
```

Comment:
To detect whether this is for an enchantment or not, let's make this an instance method:

```suggestion
Hooks.on("renderActiveEffectConfig", (app, html, data) => app.document.onRenderActiveEffectConfig(html));
```